docs/EN_US/ECLLanguageReference/ECLR_mods/Basics-Constants.xml

```diff
@@ -134,20 +134,27 @@ MyUnicodeString3 := U'abcd\u00EB'; // becomes 'abcdë'«'</programlisting>
     INTEGER<indexterm>
         <primary>INTEGER</primary>
       </indexterm> (see <emphasis role="bold">Value Types</emphasis>). Integer
-    constants may be decimal, hexadecimal, or binary values.
+    constants may be decimal, unsigned, hexadecimal, or binary values.
     Hexadecimal<indexterm>
         <primary>Hexadecimal</primary>
       </indexterm> values are specified with either a leading "0x" or a
     trailing "x" character. Binary values<indexterm>
         <primary>Binary values</primary>
       </indexterm> are specified with either a leading "0b" or a trailing "b"
-    character.</para>
+    character. Decimal values <indexterm>
+        <primary>Decimal values</primary>
+      </indexterm>are specified with trailing "d" character. Unsigned values
+    <indexterm>
+        <primary>Unsigned values</primary>
+      </indexterm>are specified with a trailing "u" character.</para>
 
     <programlisting>MyInt1  := 10;     // value of MyInt1 is the INTEGER value 10
 MyInt2  := 0x0A;   // value of MyInt2 is the INTEGER value 10
 MyInt3  := 0Ax;    // value of MyInt3 is the INTEGER value 10
 MyInt4  := 0b1010; // value of MyInt4 is the INTEGER value 10
 MyInt5  := 1010b;  // value of MyInt5 is the INTEGER value 10
+MyDec   := 10d     // value of MyDec is the DECIMAL value 10
```

Comment:
It might be clearer to use a number with a fraction e.g. 3.14159265358979323846D  (this would lose accuracy if stored as a real).